applications/IgaApplication/tests/iga_test_factory.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
do we still need this? we can also delete it later!

Reply:
I actually don't know. It used to be standard. Isn't it needed anymore @philbucher ?

Reply:
we do no longer need this, Python2 support is dropped :D

Reply:
alright. Thanks. I removed it.